includes/payment-methods/class-wc-stripe-upe-payment-gateway.php

```diff
@@ -276,12 +276,11 @@ private function get_enabled_payment_method_config() {
 	 * @return string[]
 	 */
 	public function get_upe_enabled_payment_method_ids() {
-		return $this->get_option(
-			'upe_checkout_experience_accepted_payments',
-			[
-				'card',
-			]
-		);
+		$capture = ! empty( $this->get_option( 'capture' ) ) && $this->get_option( 'capture' ) === 'yes';
```

Comment:
I think maybe this check should be moved to `get_upe_enabled_at_checkout_payment_method_ids()`. Which would still allow the payment methods to appear as enabled in the admin. It would require modifying WC_Stripe_UPE_Payment_Method::is_enabled_at_checkout(). 

It's fine to leave the check here too, I guess the real question is if card is the only payment methods that will ever support manual capture.

Reply:
I think it's okay to have them disabled in the admin because that reflects what the customers see in the checkout. On the other hand, that can be confusing for the merchant that "will not be able" to enable other payment methods.

I'm not sure if that's covered in the new settings, but we can maybe add some sort of help note/tooltip in the option to warn the merchant that disabling the automatic capture will restrict the payment methods to card only or something in that direction.